.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -15,14 +15,14 @@ jobs:
           PR_NUMBER: ${{ github.event.number }}
         name: "Set milestone on PR based on base branch"
         run: |
-          readarray -t milestones < <(gh api /repos/brave/brave-core/milestones -q '.[] | .title')
-          if [ "$GITHUB_BASE_REF" = "master" ]; then
+          readarray -t milestones < <(gh api /repos/brave/brave-core/milestones -q ".[] | .title")
```

Comment:
`readarray` and `mapfile` are the same thing, so we might want to use just one name

Reply:
Ok, preferring readarray since it sounds clearer